dev.planx-pla.net/manifest.json

```diff
@@ -36,7 +36,7 @@
     "hostname": "dev.planx-pla.net",
     "revproxy_arn": "arn:aws:acm:us-east-1:707767160287:certificate/c676c81c-9546-4e9a-9a72-725dd3912bc8",
     "dictionary_url": "https://s3.amazonaws.com/dictionary-artifacts/datadictionary/develop/schema.json",
-    "portal_app": "dev",
+    "portal_app": "gitops",
```

Comment:
you need this to tell portal to use the gitops.json from manifest as portal config file